pages/login.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: connor
+ * Date: 18/07/2016
+ * Time: 02:12
+ */
+
+define('BASE_URL', 'http://vagrant.learn.jquery.com/wp-admin/admin.php?page=jquery-q-and-a');
```

Comment:
@AurelioDeRosa 

Do you think we should update the config file to include the url and plugin name?